source/boundary_temperature/dynamic_core.cc

```diff
@@ -693,30 +693,12 @@ namespace aspect
                 if (cell->face(f)->boundary_id() == CMB_id)
                   {
                     fe_face_values.reinit (cell, f);
+
+                    // Repopulate MaterialInputs, set use_strain_rates to false
+                    in.reinit(fe_face_values, cell, this->introspection(), this->get_solution(), false);
```

Comment:
I prefer writing this as
```suggestion
                    in.reinit(fe_face_values, cell, this->introspection(), this->get_solution(), /* use_straim_rates = */ false);
```
Then you can delete the comment above.

Reply:
I'm renaming this argument to `compute_strain_rates` in #4267. Might want to use that naming scheme.